blog/2022/12-16-resoto-aws-permissions-rollout/index.md

```diff
@@ -0,0 +1,123 @@
+---
+authors: [lukas]
+tags: [inventory, metrics, graph, aggregation]
+image: ./img/banner-social.png
+---
+
+# Resoto AWS Permissions Rollout
+
+Each version of Resoto programmatically generates the specific IAM permissions it requires to collect and optionally to manipulate AWS resources. Manually maintaining these permissions is a tedious task, especially when you have multiple AWS accounts. This is why we have created a Cloudformation Stack that automatically creates the `ResotoAccess` role with the correct permissions and creates a trust that allows a specified AWS account to assume this role.
+
+This stack can be deployed in a single account or organization-wide as a Cloudformation StackSet.
+
+The source code for these stacks can be found in the [resoto-cf](https://github.com/someengineering/resoto-cf) Github repository.
```

Comment:
Let's mention explicitly somewhere:
- this stack only creates a role and permissions
- it will not install Resoto. Link to the Resoto Template install

Reply:
Ah, and I just realized: I guess there are people like me that never used a stack set and don't know what it is good for. Consider adding a sentence about it.